test/trial_scheduler_test.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import unittest
+import os
+
+from ray.tune.result import TrainingResult
+from ray.tune.trial import Trial
+from ray.tune.trial_scheduler import MedianStoppingRule, TrialScheduler
+
+
+def result(t, rew):
+    return TrainingResult(time_total_s=t, episode_reward_mean=rew)
+
+
+class EarlyStoppingSuite(unittest.TestCase):
```

Comment:
Can we also cover `time_attr='time_total_s'`, `reward_attr='episode_reward_mean'` initializer parameters?

Reply:
Done